wordlist.txt

```diff
@@ -60,3 +60,12 @@ VNET
 TKG
 NSXT
 autoscaler
+VYOS
```

Comment:
All of these wordlist items are only required because you have codeblocks inside of blockquotes. We are not using blockquotes elsewhere...can we just remove the blockquotes?

Reply:
sorry for the delay; sure thing!